canal/config.go

```diff
@@ -46,6 +46,16 @@ type Config struct {
 	HeartbeatPeriod time.Duration `toml:"heartbeat_period"`
 	ReadTimeout     time.Duration `toml:"read_timeout"`
 
+	// IncludeTableRegex or ExcludeTableRegex should contain database name
+	// Only a table match IncludeTableRegex and dismatch ExcludeTableRegex will be processed
+	// eg, IncludeTableRegex : ".*\\.canal", ExcludeTableRegex : "mysql\\..*"
+	//     this will include all database's 'canal' table, except database 'mysql'
+	// If IncludeTableRegex is empty, canal will not check IncludeTableRegex
+	// If ExcludeTableRegex is empty, canal will not check ExcludeTableRegex
+	// Default IncludeTableRegex and ExcludeTableRegex are empty, this will include all tables
+	IncludeTableRegex string `toml:include_table_regex`
```

Comment:
how to support many different tables? 

I think this should be a `[]string`

Reply:
done